src/coreclr/tools/Common/TypeSystem/Common/TypeFlags.cs

```diff
@@ -57,10 +57,11 @@ public enum TypeFlags
         HasFinalizer         = 0x2000,
 
         IsByRefLike            = 0x04000,
-        AttributeCacheComputed = 0x08000,
+        IsInlineArray          = 0x08000,
         IsIntrinsic            = 0x10000,
+        AttributeCacheComputed = 0x20000,
```

Comment:
why are the flags switched here? Why cant IsInlineArray be `0x20000`?

Reply:
I've inserted the new flag next to related flags. I could take the next flag, but that could be confusing.
AttributeCacheComputed is set when the few flags above are computed - since they are keyed of attributes.

Reply:
I think it looks nicer if AttributeCacheComputed comes after the flags that it protects. 
I do not think these flags are persisted. This is a part of compile-time type system, unless I am mistaken.